### PR TITLE
Dob certificate occupancy

### PR DESCRIPTION
**Pull request by steve52**
Issue https://github.com/nycdb/nycdb/issues/247

- I added `venv/` to `.gitgnore` because I created it in my nycdb directory. Seems reasonable, but if folks would prefer to leave it out and people create their venv's somewhere else, I'm down with that.

- This is my first added dataset, so still learning the conventions and such. Would love any feedback on the fields and their types. I included all of them, but maybe I should skip some?

- In the data I noticed that `BinNumber` always exists, but `BIN` is sometimes missing. And they're almost always the same number, but sometimes different. Anyone know what's up with that? Just curious.

### CODE REVIEW

**Comment by aepyornis** [comment]
looks good to me, thanks @steve52. not sure why travis is stalled here...

No idea what's up with the BinNumber vs Bin issue but I think others in HDC have been somewhat successful in reporting data problems to nycopendata.

**Comment by austensen** [comment]
@aepyornis  Tracis CI builds have been disabled because of some change to the plans. It looks like we have tons of credits, but just wanted to check with you before turning things back on. is that ok? https://app.travis-ci.com/organizations/nycdb/plan?anchor=oss-consumption

**Comment by aepyornis** [comment]
ah, sure let's re-enable it. If travis no longer gives enough free credits to open source projects, we can try to switch to github actions instead.

**Comment by steve52** [comment]
Regarding the BinNumber vs Bin thing.... should I report it to open data nyc? What's the process for that typically. Is it sending them a note through the form on their Contact Us page? 

Would we want to wait for a response before merging this? Or do we typically just move forward with the dataset and then adjust it in the future if they fix the issue?

File: src/nycdb/datasets/dob_certificate_occupancy.yml
Comment (austensen) [inline src/nycdb/datasets/dob_certificate_occupancy.yml:31]: Based on what I was saying about the geocoding, I think we should exclude these two columns, and instead make a BBL column from the boro-block-lot (like we do for other datasets without it).

File: src/nycdb/sql/dob_certificate_occupancy.sql
Comment (austensen) [inline src/nycdb/sql/dob_certificate_occupancy.sql:1]: Since we are keeping only `binnumber`, can we rename it in the sql here to just `bin` for consistency with other datasets - and add indexes for `bin`, `jobnumber` as well?

**Review by austensen — APPROVED** [review]
Thanks for working on this. Just made a couple minor suggestions, but this all looks great!

**Comment by austensen** [comment]
I was just looking around on open data and found that there is a second version of the CofO data that I think we should add as a second table part of this dataset. DOB has been slowly transitioning over to a new internal system for managing all their operations - DOB NOW - and annoyingly they now have an old and new version of many of their datasets. 

https://data.cityofnewyork.us/Housing-Development/DOB-NOW-Certificate-of-Occupancy/pkdm-hqz6

**Comment by steve52** [comment]
Closing and replacing with https://github.com/nycdb/nycdb/pull/282

**Pull request closed without merging**
